JoinColony/colonyCDapp
# fix: changed failed finalize step content

You should not be able to finalize a failed action.
You should be able to return stakes to users.

## Testing

- Ensure you have the Reputation extension installed with the testing configuration.
- Create an action using the Reputation decision method.
- Don't stake at all on the action.
- Allow the time to pass and the action to fail.

Resolves [https://github.com/JoinColony/colonyCDapp/issues/2165](https://github.com/JoinColony/colonyCDapp/issues/2165)
